src/platform.rs

```diff
@@ -12,6 +12,8 @@ cfg_if::cfg_if! {
         }
     } else if #[cfg(blake3_neon)] {
         pub const MAX_SIMD_DEGREE: usize = 4;
+    } else if #[cfg(blake3_wasm32_simd)] {
+        pub const MAX_SIMD_DEGREE: usize = 8;
```

Comment:
Should this be 4? Either way, `MAX_SIMD_DEGREE` and `MAX_SIMD_DEGREE_OR_2` should definitely be equal if the former is 2 or greater.